core/.js/src/test/scala/io/udash/testing/TestViewFactory.scala

```diff
@@ -25,7 +25,7 @@ class TestView extends ContainerView {
   }
 }
 
-class TestFinalView extends FinalView {
+class TestFinalView extends View {
```

Comment:
There are no final / container states now.  Any other improvements you suggest here?

Reply:
It's much better now. I'd suggest to redefine `TestViewFactory` slightly because:
 1. generic type `T` is not used,
 2. container `TestView` is returned for `ErrorState`, marked as `FinalTestState`.

```scala
object TestViewFactory extends ViewFactory[RootState] {
  val view = new TestView
  val presenter = new TestPresenter[RootState]

  override def create(): (View, Presenter[RootState]) = (view, presenter)
}
```